code/modules/surgery/organs/organ_external.dm

```diff
@@ -320,6 +321,8 @@ This function completely restores a damaged organ to perfect condition.
 	for(var/obj/item/organ/internal/current_organ in internal_organs)
 		current_organ.rejuvenate()
 
+	for(var/obj/item/organ/external/EO in contents)
```

Comment:
Chop off a left leg--it contains the left foot.

This was a problem we ran into last night; even when you reset the germ level and poured mito on an external limb, then re-attached it, the child limb would still be "dead".